stubs/AMQPExchange.php

```diff
@@ -211,7 +211,7 @@ public function setArguments(array $arguments)
      *                       AMQP_DURABLE, AMQP_PASSIVE
      *                       (and AMQP_DURABLE, if librabbitmq version >= 0.5.3)
      *
-     * @return boolean True on success or false on failure.
+     * @return void
```

Comment:
This is a hollywar topic, but in the face of [Void Return Type](https://wiki.php.net/rfc/void_return_type), shouldn't we just drop any `@return` tag at all if method/function just doesn't (or shouldn't) explicitly return any result? I see that `void` clearly say that one should not relay on function result while not `@return` tag is not clear at this point.